src/DiffChecker.js

```diff
@@ -153,6 +156,31 @@ export class DiffChecker {
     return false
   }
 
+  /**
+   * Function to check if any new added file dose not do a full coverage
+   */
+  checkIfNewFileNotFullCoverage() {
+    if (!this.checkNewFileFullCoverage) return false
+    const keys = Object.keys(this.diffCoverageReport);
+    return keys.some((key) => {
+      const diffCoverageData = this.diffCoverageReport[key];
+      const coverageParts = Object.values(diffCoverageData);
+      // No old coverage found so that means we added a new file coverage
+      const newFileCoverage = coverageParts.every((coverageData) => coverageData.oldPct === 0);
+      return newFileCoverage && this.checkIfNewFileNotFullCoverageOnAnyPart(coverageParts) && this.checkOnlyChangedFiles(key);
+    });
+  }
+
+  
+  /**
+   * Function to check whether any part dose not do a full coverage
+   * @param  {} coverageParts
+   * @param  {} {return boolen}
```

Comment:
This is not the correct JSDoc syntax. See the [docs site](https://jsdoc.app/tags-returns.html) for more details.
```suggestion
   * @returns  {boolean}
```